src/ustreamer/device.c

```diff
@@ -338,6 +345,9 @@ int us_device_grab_buffer(us_device_s *dev, us_hw_buffer_s **hw) {
 			}
 			GRABBED(new) = true;
 
+			if (dev->capture_type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE) 
+				new.bytesused = new.m.planes[0].bytesused;
```

Comment:
should we add bytes of the other plane which index > 0?

Reply:
I'am not sure about this. I prefer to stick with 1 plane for now, until any user has device with 2 or more planes to test it.

Reply:
theorically, YUV format will have a plane count >1, but as you see my device is ok with 1 here, so.....

Reply:
I found somewhere some formats can use multiple planes(e.g. 1 plane for Y and 1 for CbCr), let's wait for an issue for this, until someone can have a device to test it.